conftest.py

```diff
@@ -6,6 +6,7 @@
 # https://github.com/gevent/gevent/issues/1016#issuecomment-328530533
 # https://github.com/spyoungtech/grequests/issues/8
 import grequests
+
```

Comment:
These are just changes due to black. If we want to remove them from this PR let me know